packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Sign up for new account with email only
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+
+Learn how to use the Embedded SDK to integrate password optional sign up in your app.
+
+**Learning outcomes**
+
+* Configure your Okta org to allow your users the option to sign up without a password
+* Enable your app to support password optional sign ups
+
+**What you need**
+
+<StackSnippet snippet="whatyouneed" />
+</br>
+
+**Sample code**
+
+<StackSnippet snippet="samplecode" />
+
+---
+
+## Overview
+
+## Update configurations
+
+Before you can start integrating password-optional sign-ups in your app, <StackSnippet snippet="setupoktaorg" inline/>. Additionally, you must enable new user sign-ups with the following steps:
```

Comment:
```suggestion
Before you can start integrating password-optional sign-up flows in your app, <StackSnippet snippet="setupoktaorg" inline/>. Additionally, you must enable new user sign-up flows with the following steps:
```

Reply:
Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435